skabase/SKALogger/test/SKALogger_test.py

```diff
@@ -98,6 +98,7 @@ def test_SetCentralLoggingLevel(self, tango_context, setup_log_test_device,
         device_details.append(targets)
         tango_context.device.SetCentralLoggingLevel(device_details)
         dev_proxy = DeviceProxy(logging_target)
+        print("Element logging level: ", dev_proxy.elementLoggingLevel)
```

Comment:
Were these printouts for debugging, or something that we should keep for the future?

Reply:
This one remained by mistake. Removed.